yarn-error.log

```diff
@@ -0,0 +1,7019 @@
+Arguments: 
```

Comment:
I forget to add this file in `.gitignore` ...

Can you add it inside the `.gitignore` file and remove this file from the pull request ? 😅